docs/_docs/faq.md

```diff
@@ -186,6 +186,33 @@ In the Zap Editor, Zapier will attempt to retrieve or create existing data to te
 
 Make sure to provide JSON-formatted sample data using the same field names as your API. The sample response data should not include any personally identifiable data, have copy that is safe for work, and be easily recognizable as sample data.
 
+When providing sample data, it's also important that it only include fields that are present every time a Zap runs. When a field is provided in the sample, it can be mapped into a field in a later action.
+
+If that field isn't available when the user's Zap runs, the action field will be empty, which can cause errors or unexpected results for users. For example, suppose your sample data looks like this:
+
+```json
+{
+  "id": 1,
+  "first_name": "Jane",
+  "last_name": "Suarez",
+  "email_address": "janesz@example.com",
+  "job_title": "Executive Director"
+}
+```
+
+A user might map the "Job Title" information into a required field in another app, such as a CRM. Then, when the Zap runs, `job_title` is only included in the live result if it happens to be available, and the data the Zap receives looks like this:
```

Comment:
I'd vote for something more precise around the object, given we are talking to the dev. Could you just use "`job_title`" all the way through here?

Reply:
I'm not sure if I understand what you're envisioning - can you share some details, or make a suggested edit, even?

Reply:
Is the concern that the dev doesn't see this as equivalent to "Job Title", so you'd rather use `job_title` throughout? I can see that, although I was hoping to bring a little bit of the user perspective here.